docs/home/messages-channels-and-schemas.md

```diff
@@ -0,0 +1,19 @@
+# Messages, Channels and Schemas
+
+MCAP organizes its data via messages, channels, and schemas.
+
+## Message
+
+The unit of communication between nodes in the pub/sub system.
+
+## Channel
+
+A stream of messages which have the same topic and type, or schema. This is often called a "topic" in pub/sub framework terminology.
```

Comment:
@esthersweon I updated this - previously it said that a channel often represents a "connection between a publisher and a subscriber", but this felt wrong to me. Pub/sub tries to abstract those connections away, and subscribers just "subscribe to a topic" rather than connecting directly to a subscriber.

Reply:
We use `topic` to refer to the string field in the Channel record which specifies the topic that a subscriber would've subscribed to. However, there may be multiple Channels corresponding to the same topic. We inherited/adapted this from ROS where they are called "connections" because they represent a publisher–subscriber connection (in the case of a bag, the connection is between the publisher and the bag recorder)

Reply:
On a more pedantic note, the proposed new wording is strange because it says that a topic is a stream of messages which have the same topic.